packages/mdc-shape/_functions.scss

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@function mdc-shape-corners($top-left, $top-right: null, $bottom-right: null, $bottom-left: null) {
```

Comment:
I think this can be rewritten like:

```
@function mdc-shape-corners($top-left, $top-right: $top-left, $bottom-right: $top-left, $bottom-left: $top-left) {
  @return (
    top-left: $top-left,
    top-right: $top-right,
    bottom-left: $bottom-left,
    bottom-right: $bottom-right
  );
}
```

I tried it in sassmeister:

<img width="963" alt="screen shot 2018-04-04 at 14 46 45" src="https://user-images.githubusercontent.com/579873/38336596-2f0d5cec-3817-11e8-8faa-237fbb87ceaf.png">

Reply:
Really good point. I'm going to remove the function entirely because there's no point when this can be done directly in default parameters.

I suspect when I first came up with this I was preoccupied by the order the defaults are applied, but they apply in the same order that the parameters are defined, so it still works.